index.html

```diff
@@ -99,23 +108,50 @@
           <svg class="header-buttons__account">
             <use href="#account"/>
           </svg>
+          <svg class="header-buttons__menu">
+            <use href="#menu"/>
+          </svg>
         </div>
       </div>
     </div>
     <h1 class="title topsection-title">Discover stays<br> to live, work or just relax</h1>
     <div class="form">
       <form class="topsection-form" action="./" method="get">
         <div class="topsection-city">
-          <label class="topsection-city__label" for="city">Your destination or hotel name</label>
-          <input class="input topsection-city__input" id="city" type="text" name="city" placeholder="New York"/>
+          <label class="topsection-city__label label" for="city">Your destination or hotel name</label>
+          <input class="input topsection-city__input" id="city" type="text" name="city" placeholder="Your destination or hotel name"/>
+          <svg class="search-icon">
+            <use href="#search"/>
+          </svg>
         </div>
         <div class="topsection-date">
-          <label class="topsection-date__label" for="date">Check-in &ndash; Check-out</label>
-          <input class="input topsection-date__input" id="date" type="date" name="date"/>
+          <label class="topsection-date-in__label label" for="check-in">Check-in</label>
+          <input class="input topsection-date-in__input" id="check-in" type="text" name="date" placeholder="Tue 15 Sept" value="Tue 15 Sept"/>
+          <div class="separator">
+            <p> &ndash; </p>
+          </div>
+          <label class="topsection-date-out__label label" for="check-out">Check-out</label>
```

Comment:
This label is wrong positioned in mobile view. I offer 2 ways to solve it: 
1. use relative units in position and select it for every type of screen width;
2. wrap every input with label into div and position label related to that div instead of div.topsection-date.